src/TextureManager.cpp

```diff
@@ -1,9 +1,11 @@
-#include <iostream>
```

Comment:
As I mentioned for `TextureManager.hpp`, I let's keep the include for `iostream` here and add one for `string`.